README.md

```diff
@@ -1,16 +1,25 @@
 ##SwiftLint for AppCode
 
-This simple plugin allows to highlight errors and warnings from SwiftLint (https://github.com/realm/SwiftLint) like this 
+This simple plugin allows to highlight errors and warnings from SwiftLint (https://github.com/realm/SwiftLint) like this:
+
 ![](http://d.pr/i/goBy/3VsjzzWL+)
 
 
 ###Installation
 
 You should already have SwiftLint installed somewhere.
- 1. Configure SwiftLint via .swiftlint.yml file, if needed
- 2. Install the plugin from the repository
+ 1. Configure SwiftLint via `.swiftlint.yml` file, if needed. See [here](https://github.com/realm/SwiftLint#configuration) fir details.
+ 
+    Note that you have to set `reporter: "xcode"`.
```

Comment:
It is default, afaik. And it looks like a good choice because it should work simultaneously with the Xcode. I believe that better comment would be something like:

`Note that plugin works with "reporter: xcode".`

Without any specifics, where to set it up, as a default or in project configuration file.